### ISSUE TITLE
Mobile: Preserve caret on drag in edit mode

### ISSUE THREAD

**Issue by raineorshine** [cybersemics/em#1387]
Tested on iOS Safari.

## Steps to Reproduce

```
- a
- b
```

1. Set the caret on `a`
2. Drag `a` below `b`

## Current Behavior

Caret is gone even though the on-screen keyboard is still up.

## Expected Behavior

Caret should be preserved on drag in edit mode.

**Comment by Zubair286** [comment]
Tested on iOS safar and chrome, unable to reproduce. Please confirm

**Comment by raineorshine** [comment]
Yes, I am no longer able to reproduce this. Maybe it was a Safari bug they fixed!

There are some cases where the keyboard closes instead of staying open, but the selection is correctly removed, so there is no longer any inconsistency, which is the most important part.
